packages/create-app/templates/default-app/docker-compose.yaml.hbs

```diff
@@ -0,0 +1,40 @@
+version: "3.4"
+services:
+  {{#if dbTypePG}}
+  # TODO: Need to figure out why databases aren't auto created
```

Comment:
A docker specific way around this would be to do the following:

```yaml
# docker-compose.yaml
services:
  database:
    # ...
    volumes:
      - ./docker/init.sql:/docker-entrypoint-initdb.d/init.sql
```

Then in `./docker/init.sql`

```sql
# docker/init.sql
CREATE DATABASE backstage_plugin_catalog;
CREATE DATABASE backstage_plugin_auth;
```

Reply:
Gotcha thanks that works! I'll add this in